src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/services/GetPersonService.kt

```diff
@@ -105,8 +104,47 @@ class GetPersonService(
   }
 
   fun getPersonFromNomis(nomisNumber: String) = prisonerOffenderSearchGateway.getPrisonOffender(nomisNumber)
-}
 
-fun isNomsNumber(id: String?): Boolean {
-  return id?.matches(Regex("^[A-Z]\\d{4}[A-Z]{2}+$")) == true
+  fun getPrisoner(hmppsId: String): Response<Person?> {
+    val prisonerNomisNumber = getNomisNumber(hmppsId)
```

Comment:
will the api only accept noms id or crn?

Reply:
just noms